tellapart/aurproxy/backends/nginx/backend.py

```diff
@@ -56,6 +56,10 @@ def __init__(self, configuration, signal_update_fn):
                                               config,
                                               default=None,
                                               required=False)
+    self._nginx_pid_path = self._load_config_item('nginx-pid-path',
```

Comment:
I'd prefer to separate the words in this configuration element name with "_" instead of "-" for the sake of consistency.